### PR TITLE
release notes for 1.1.0-M1

### PR DESCRIPTION
**Pull request by pjfanning**
relates to https://github.com/apache/incubator-pekko/issues/945

it would be good if other contributors could check if any important updates have not been listed

### CODE REVIEW

File: docs/src/main/paradox/release-notes/releases-1.1.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+* The pekko-protobuf jar is no longer published. The pekko-protobuf-v3 jar is still published ([PR489](https://github.com/apache/incubator-pekko/pull/489))
+
+### Other Changes
+
+* Scala 2 inline optimizer has been enabled
+* Classic Remoting was updated to use Netty 4 instead of Netty 3 ([PR643](https://github.com/apache/incubator-pekko/pull/643))
+* Support configuration for Jackson Recycler Pool ([PR1192](https://github.com/apache/incubator-pekko/pull/1192))
+* pekko-multi-node-testkit was changed to use Netty 4 instead of Netty 3 ([PR539](https://github.com/apache/incubator-pekko/pull/539))
+* add junit5 support to pekko-testkit-typed ([PR751](https://github.com/apache/incubator-pekko/pull/751))
+* Some performance changes in the Stream code ([PR48](https://github.com/apache/incubator-pekko/pull/48), [PR49](https://github.com/apache/incubator-pekko/pull/49), [PR278](https://github.com/apache/incubator-pekko/pull/278), [PR363](https://github.com/apache/incubator-pekko/pull/363), [PR408](https://github.com/apache/incubator-pekko/pull/408), [PR983](https://github.com/apache/incubator-pekko/pull/983), [PR1027](https://github.com/apache/incubator-pekko/pull/1027))
```
Comment (He-Pin) [inline docs/src/main/paradox/release-notes/releases-1.1.md:47]: https://github.com/apache/incubator-pekko/pull/252 is a behavior change, @mdedetrich , should be included into the release note too.

File: docs/src/main/paradox/release-notes/releases-1.1.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Release notes for Apache Pekko 1.1.0-M1. See [GitHub Milestone](https://github.com/apache/incubator-pekko/milestone/2?closed=1) for fuller list of changes.
+
+### 1.0.x changes
+
+Apache Pekko 1.1.0-M1 contains all the changes that have been released in the @ref:[1.0.x releases](releases-1.0.md), with the exception of the changes to support interacting with Akka remote and cluster instances ([PR765](https://github.com/apache/incubator-pekko/pull/765), [PR1112](https://github.com/apache/incubator-pekko/pull/1112)).
+
+### Bug Fixes
+
+* in TlsActor, flush data to user only after handshake has finished ([PR1128](https://github.com/apache/incubator-pekko/pull/1128))
```
Comment (He-Pin) [inline docs/src/main/paradox/release-notes/releases-1.1.md:13]: https://github.com/apache/incubator-pekko/pull/1156

File: docs/src/main/paradox/release-notes/releases-1.1.md (hunk 1 above)
Comment (pjfanning) [reply docs/src/main/paradox/release-notes/releases-1.1.md:47]: adding now

File: docs/src/main/paradox/release-notes/releases-1.1.md (hunk 2 above)
Comment (pjfanning) [reply docs/src/main/paradox/release-notes/releases-1.1.md:13]: adding 1156 - 1088 is listed as a fix in 1.0.x and this page starts by saying all 1.0.x changes are included in 1.1.0-M1

File: docs/src/main/paradox/release-notes/releases-1.1.md (hunk 1 above)
Comment (pjfanning) [reply docs/src/main/paradox/release-notes/releases-1.1.md:50]: done

**Comment by mdedetrich** [comment]
Let's wait until merging since https://github.com/apache/incubator-pekko/pull/981 is on the horizon of being completed

**Comment by pjfanning** [comment]
> Let's wait until merging since #981 is on the horizon of being completed

We don't typically merge these until after the release is approved. We can link this PR in the vote email.

**Comment by mdedetrich** [comment]
> We don't typically merge these until after the release is approved. We can link this PR in the vote email.

Yes I know, just leaving it here as a note so we don't forget it in the notes

File: docs/src/main/paradox/release-notes/releases-1.0.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+# Release Notes (1.0.x)
+
+## 1.0.3-M1
+This is milestone release and is aimed at testing some new support for users who want to add Pekko nodes to an existing Akka cluster. This support is experimental. This release should not be used in production.
+
+### Bug Fixes
+
+* Fix StackOverflowError in ByteStringBuilder.addAll ([PR903](https://github.com/apache/pekko/pull/903))
+* Fix issue with possible int overflow in snapshot interval calculation ([PR1088](https://github.com/apache/pekko/pull/1088))
```
Comment (nvollmar) [inline docs/src/main/paradox/release-notes/releases-1.0.md:9]: Maybe mention #1183
This is quite an annoying bug when running larger Pekko clusters in k8s.

**Comment by pjfanning** [comment]
The release vote has passed so I need to get the docs updated as part of the release.

**Comment by pjfanning** [comment]
I need to get this merged so that I can announce the release.

**Review by raboof — APPROVED** [review]

**Review by danischroeter — COMMENTED** [review]
lgtm

**Pull request merged**